src/test/java/com/powsybl/openloadflow/ac/GeneratorTargetVoltageInconsistencyTest.java

```diff
@@ -45,7 +45,7 @@ void localTest() {
                 .setMinP(0)
                 .setMaxP(200)
                 .setTargetP(100)
-                .setTargetV(413)
+                .setTargetV(23)
```

Comment:
Could you explain why you changed unit tests?

Reply:
Because the previous test was not real with two generators connected to a 20 kV voltage level and with target V around 400 kV, without any remote regulating terminal.